src/ralph/ssl_certificates/management/commands/update_dns_service_env_from_dnsaas.py

```diff
@@ -8,7 +8,10 @@
 
 
 def checking_type(value):
-    if value['type'] in {'CNAME', 'PTR'}:
+    domain = ''
+    if value['type'] in {'MX', 'TXT'}:
+        pass
```

Comment:
```
if value['type'] in {'MX', 'TXT'}:
    domain = ''
```

Reply:
Fair point

Reply:
I think we definitely need and comment here - why is this change now required?

Reply:
Bacause all our MX and TX records have the same service (email) and this this two records gives us false data

Reply:
IMO it would be good to add this comment in code for future generations ;)